azure-pipelines.yml

```diff
@@ -0,0 +1,151 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - release/6.x
+    - release/9.x
+
+pr:
+  branches:
+    include:
+    - master
+
+variables:
+- ${{ if ne(variables['System.TeamProject'], 'public') }}:
+  - group: DotNet-VSTS-Bot
+  - group: DotNet-Blob-Feed
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+  # Variables used by arcade to publish packages
+  - name: _DotNetFeedUrl
+    value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+  - name: _DotNetValidationArtifactsCategory
+    value: .NETCore
+  - name: _DotNetArtifactsCategory
+    value: .NETCore
+
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      enablePublishBuildArtifacts: false
+      enablePublishBuildAssets: false
+      enablePublishUsingPipelines: false
+      variables:
+        - _BuildConfig: Release
+      jobs:
+
+      ############ LINUX BUILD ############
+      - job: Build_Linux
+        displayName: Linux
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            x64:
+              poolname: Hosted Ubuntu 1604
+              assetManifestOS: linux
+              assetManifestPlatform: x64
+              installDependencies: true
+        pool:
+          name: $(poolname)
+        steps:
+        - bash: |
+            set -ex
+            git clean -ffdx
+            git reset --hard HEAD
+          displayName: 'Clean up working directory'
+
+        - bash: |
+            set -ex
+            sudo apt update
+            sudo apt -y install nuget libgdiplus build-essential libtool libtool-bin cmake gettext bc
```

Comment:
pretty sure we don't need nuget/libgdiplus/bc here? not sure about gettext and libtool

Reply:
You're right.. copy-pasta from Mono's yml